test/i965_avce_context_test.cpp

```diff
@@ -65,6 +65,9 @@ class AVCEContextTest
             ||IS_GEN10(i965->intel.device_info))
             is_gen9 = true;
 
+        if (VAProfileJPEGBaseline == profile)
```

Comment:
This is unnecessary since all these test cases deal with H264 profiles only.

Reply:
thanks